web/static/js/reducers/stage.js

```diff
@@ -1,5 +1,7 @@
 const stage = (state = "idea-generation", action) => {
   switch (action.type) {
+    case "SET_INITIAL_STATE":
+      return action.initialState.stage
```

Comment:
@travis is it redundant to `SET_INITIAL_STATE` if we're already providing the `state` param with a default argument of `idea-generation`?